src/router/index.js

```diff
@@ -97,7 +103,8 @@ export default new Router({
 			}),
 		},
 		{
-			path: '/tags/:path*',
+			// path: '/tags/:path*',
+			path: tagsPath,
```

Comment:
The router path should stay `'/tags/:path*'`

Reply:
you can add 
```
beforeEnter() {
                        if (!areTagsInstalled) {
				window.open(generateUrl('/apps/photos'))
			},
},
```

Reply:
With this change the tags route isn't working any more

Reply:
@Mikescops
any ideas what I can do? beforeEnter code is unfortunately not working.

Reply:
let me checkout you're code locally